mmv1/third_party/validator/tests/data/example_cloud_run_service_iam_binding.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "name": "//run.googleapis.com/cloudrun-srv",
```

Comment:
Changing this is expected to impact all iam resources generated so far...

Reply:
yeah, it would. This is definitely a bug, though. It looks like there are also other IAM resources that are impacted by this.

I'll open a separate bug for fixing this generation & merge this for now.

Reply:
b/211764514